scripts/build/Jenkins/Jenkinsfile

```diff
@@ -414,7 +405,13 @@ def PreBuildCommonSteps(Map pipelineConfig, String snapshot, String repositoryNa
         }
     }
 
+    def workspace_retry = 5
```

Comment:
Is this issue limited to the workspace directory specifically or do we need to add this retry for other subdirectories created using dir().

Reply:
Just the root workspace specifically. It seems the issue is that the EBS volume may not be immediately available for writes once mounted. Once this is available, then the rest of the subfolders can be created.